packages/valory/skills/transaction_settlement_abci/tests/test_rounds.py

```diff
@@ -107,6 +107,14 @@ def get_participants() -> FrozenSet[str]:
     return frozenset([f"agent_{i}" for i in range(MAX_PARTICIPANTS)])
 
 
+def get_uniform_payloads(round_cls, **kwargs):  # AbstractRound
+    """Get uniform payloads"""
```

Comment:
Please update this docstring. Not obvious what it is doing without reading the code.